packages/devtools_app/lib/src/screens/memory/panes/diff/diff_pane.dart

```diff
@@ -161,22 +161,35 @@ class _SnapshotListTitle extends StatelessWidget {
     required this.item,
     required this.selected,
   }) : super(key: key);
+
   final DiffListItem item;
+
   final bool selected;
 
   @override
   Widget build(BuildContext context) {
-    return AnimatedBuilder(
-      animation: item,
-      builder: (context, child) => Row(
+    final theItem = item;
+    return ValueListenableBuilder<bool>(
+      valueListenable: theItem.isProcessing,
+      builder: (_, isProcessing, __) => Row(
         children: [
-          _SelectionBox(selected: selected),
           const SizedBox(width: denseRowSpacing),
-          if (item.isProcessing) const _ProgressIndicator(),
-          if (item.isProcessing) const SizedBox(width: denseRowSpacing),
-          Expanded(
-            child: Text(item.name, overflow: TextOverflow.ellipsis),
-          ),
+          if (theItem is SnapshotListItem)
+            Expanded(
+              child: Text(theItem.name, overflow: TextOverflow.ellipsis),
+            ),
+          if (theItem is InformationListItem) ...[
+            const Expanded(
+              child: Text('Snapshots', overflow: TextOverflow.ellipsis),
+            ),
+            const SizedBox(width: denseRowSpacing),
+            const Text('\u{24D8}'),
```

Comment:
why are we using this instead of an Icon?

Reply:
Looks much nicer and less widgets.